plsync/sites.py

```diff
@@ -79,6 +80,7 @@
     makesite('atl05','67.106.215.192', '2610:18:111:c002::',   'Atlanta_GA', 'US', 33.636700, -84.428100, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos'),
     makesite('atl06','70.42.177.64',   '2600:c0b:2002:5::',    'Atlanta_GA', 'US', 33.636700, -84.428100, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos'),
     makesite('ath03','193.201.166.128', '2001:648:25e0::',     'Athens', 'GR', 37.936400, 23.944400, user_list, count=4, v6gw='2001:648:25e0::129', nodegroup='MeasurementLabCentos'),
+    makesite('bcn01','91.213.30.192',   '2001:67c:137c:5::',   'Barcelona', 'ES', 41.2974, 2.0833, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos'),
```

Comment:
This isn't a huge deal, but every other lat/lon in the file is specified to 6 decimal places. Seems like ideally we'd follow that same pattern here.

Side note: I'm going to change the note that Jordan had in the Deployment doc, removing the part of only specifying the lat/lon to 4 decimals places, and change it to specify 6 places.